content/blog/usethis-2-0-0/index.Rmd

```diff
@@ -212,7 +204,7 @@ This family has gained a couple of new functions and some improvements to existi
 * `pr_resume()` resumes work on an existing local PR branch. It can be called
   argument-less, to select a branch interactively.
   
-    ```{r, eval = FALSE}
```

Comment:
When we do this, should we add the `r` language identifier to the opening code fence?

Reply:
I don't think so, because it's not valid R code.